__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>realted -&gt; related. Not sure related  is the best word... it's more than related to those features... it IS those features.</p><p> </p><p>Might be better to call df.describe after loading the data rather than naming all of the features manually here. </p><p> </p><p>More details of the dataset: number of examples, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>